.travis.yml

```diff
@@ -0,0 +1,23 @@
+sudo: false
+language: python
+
+cache:
+  directories:
+    - "$HOME/google-cloud-sdk/"
```

Comment:
It basically zips it up and stores it somewhere on success, then retrieves that the next time you run a build. see: https://docs.travis-ci.com/user/caching/